drivers/espi/Kconfig.xec_v2

```diff
@@ -145,4 +145,18 @@ config ESPI_SAF_INIT_PRIORITY
 	help
 	  Driver initialization priority for eSPI SAF driver.
 
+config ESPI_PERIPHERAL_KBC_IBF_EVT_DATA
+	bool "KBC event data format in IBF"
+	default n
+	depends on ESPI_PERIPHERAL_8042_KBC
+	help
+	  Enable espi_evt_data_kbc format for encoding event in KBC IBF ISR
+
+config ESPI_PERIPHERAL_KBC_OBE_CBK
+	bool "KBC OBE Callback"
+	default n
+	depends on ESPI_PERIPHERAL_8042_KBC
```

Comment:
since you have multiple `depends on ESPI_PERIPHERAL_8042_KBC`, consider wrapping these in `if` instead